build/NuGetAdditionalFiles/RunCsc

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
```

Comment:
Oh, I'm not worried about breaking people. This package will only end up on MyGet right now.